v21.1/backup.md

```diff
@@ -23,42 +23,52 @@ You can also back up:
+-----------+-------------
+`targets` | Back up the listed [targets](#targets).
+`subdirectory` | The name of the specific subdirectory (e.g., `2021/03/23-213101.37`) where you want to add an incremental backup. To view available subdirectories, use [`SHOW BACKUPS IN destination`](show-backup.html). If the `subdirectory` is not provided, a full backup will be created in the collection using a date-based naming scheme (i.e., `<year>/<month>/<day>-<timestamp>`).<br><br>**Warning:** If you use an arbitrary `STRING` as the subdirectory, a new full backup will be created, but it will never be shown in `SHOW BACKUPS IN`. We do not recommend using arbitrary strings as subdirectory names.
+`LATEST` | Append an incremental backup to the latest completed full backup's subdirectory.
 `destination` | The URL where you want to store the backup.<br/><br/>For information about this URL structure, see [Backup File URLs](#backup-file-urls).
 `timestamp` | Back up data as it existed as of [`timestamp`](as-of-system-time.html). The `timestamp` must be more recent than your cluster's last garbage collection (which defaults to occur every 25 hours, but is [configurable per table](configure-replication-zones.html#replication-zone-variables)).
-`full_backup_location` | Create an incremental backup using the backup stored at the URL `full_backup_location` as its base. For information about this URL structure, see [Backup File URLs](#backup-file-urls).<br><br>**Note:** After a full backup for an explicit list of tables and/or databases, it is not possible to create an incremental backup if one or more tables were [created](create-table.html), [dropped](drop-table.html), or [truncated](truncate.html). In these cases, you must create a new [full backup](#full-backups) before more incremental backups can be created. To avoid this, [backup the cluster](#backup-a-cluster) instead of the explicit list of tables/databases.
-`incremental_backup_location` | Create an incremental backup that includes all backups listed at the provided URLs. <br/><br/>Lists of incremental backups must be sorted from oldest to newest. The newest incremental backup's timestamp must be within the table's garbage collection period. <br/><br/>For information about this URL structure, see [Backup File URLs](#backup-file-urls). <br/><br/>For more information about garbage collection, see [Configure Replication Zones](configure-replication-zones.html#replication-zone-variables).
-`kv_option_list` | Control the backup behavior with a comma-separated list of [these options](#options).
+`backup_options` | Control the backup behavior with a comma-separated list of [these options](#options).
 
-{{site.data.alerts.callout_info}}
-The `BACKUP` statement cannot be used within a [transaction](transactions.html).
-{{site.data.alerts.end}}
+### Targets
+
+Target                             | Description
+-----------------------------------+-------------------------------------------------------------------------
```

Comment:
~Suggest N/A here - this had me asking "what is 'Empty Target'?"
~Suggest writing out the example link, e.g., "For an example of a full cluster backup, see [Backup a Cluster](xxx)"